### ISSUE TITLE
Annotation check orc2

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#959]
@mah11 
I was looking at the list  of genes annotated to
"inviable elongated multinucleate aseptate vegetative cell"...which I believe is a "cell cycle arrested at cytokinesis" phenotype. This would hold for all the annotations except orc2
http://www.ncbi.nlm.nih.gov/pubmed?term=8552194

I had a quick look, and I  think were they look multinucleate at later time points, they are probably 'cutting', or attempting but not completing chromosome segregation? so they are blocked at metaphase/anaphase transition...could you have a look and se if you agree?

v

**Comment by mah11** [comment]
I have changed annotations from FYPO:0002024 to  FYPO:0001490, and from FYPO:0001271 to FYPO:0003241.

Although the FACS profile is kinda suggestive, I'm not totally confident they're cut because I can't make out the septum in those old micrographs. (That also helps the case for not using any 'inviable cell yada' term that mentions the septum.) So the cells could cut, or have a septum form after the nuclear DNA divides unequally but completely, or not have a septum form. I can't tell, and I don't remember why I picked overly-specific terms. I'll leave the FYPO:0000229 annotation as is (parent of cut).

I can see no way to tell if they're blocked at the metaphase/anaphase transition. Given that it's an ORC mutant, G2/M seems at least as likely.
